### ISSUE TITLE
Enhancement: Misuse of `&`?

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#169]
Some of the less SCSS savvy devs on my team have done this:

Bad:

``` scss
.foo {
   & > .bar {
      color: red;
   }
}
```

Good:

``` scss
.foo {
   > .bar {
      color: red;
   }
}
```

Obviously, this has no affect on the generated CSS (right?), but I would like to eliminate this "misuse" of the `&`.

My question is should this be a separate linter, or integrated into `MergeableSelector`?
